index.html

```diff
@@ -601,9 +601,32 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 or local resources.</li>
 <li>The time immediately after the user agent finish establishing
 the connection to the server to retrieve the resource, otherwise.
-<p>The returned time MUST include the time interval to establish
-the transport connection, as well as other time intervals such as
-SSL handshake and SOCKS authentication.</p>
+
+<p>The returned time MUST include the time interval to establish the transport
+connection, as well as other time intervals such as SOCKS authentication. It
+MUST include the time interval to complete enough of the TLS handshake to
+request the resource.</p>
+
+<ul>
+  <li>If the user agent used TLS False Start [[RFC7918]] for this connection,
+    this interval does not include the time needed to receive the server's
+    Finished message.</li>
+  <li>If the user agent sends the request with early data [[RFC8470]] without
+    waiting for the full handshare to complete, this interval does not include
```

Comment:
Nit: is 'does not' normative? Or perhaps change to 'should not' or 'must not'.

Reply:
"MUST NOT" makes sense